sw/device/lib/dif/dif_entropy_src_unittest.cc

```diff
@@ -214,5 +214,23 @@ TEST_F(ReadTest, ReadOk) {
   EXPECT_EQ(got_word, expected_word);
 }
 
+class ReadFifoDepthTest : public EntropySrcTest {};
+
+TEST_F(ReadFifoDepthTest, EntropyBadArg) {
+  uint32_t depth;
+  EXPECT_DIF_BADARG(dif_entropy_src_get_fifo_depth(nullptr, &depth));
+}
+
+TEST_F(ReadFifoDepthTest, DepthBadArg) {
+  EXPECT_DIF_BADARG(dif_entropy_src_get_fifo_depth(&entropy_src_, nullptr));
+}
```

Comment:
nit: (for the future) you can test all bad args for a single function in a single test case.

Reply:
I usually use this approach, but in this case I split to keep the consistency with the other tests in this file.